.release-please-manifest.json

```
@@ -0,0 +1,3 @@
+{
+  ".": "3.1.1"
```

```suggestion
  ".": "3.2.0"
```
I guess this is the current version of the project using release-please?

---

version has set to 3.2.0